kinder/pkg/cluster/manager/actions/waiter.go

```diff
@@ -307,7 +309,7 @@ func kubeletHasRBAC(major, minor uint) func(c *status.Cluster, n *status.Node) b
 				"-n=kube-system",
 				"-o=jsonpath='{.metadata.name}'",
 			)
-			if output1 != "" && output2 != "" {
+			if strings.Trim(output1, "'") == name1 && strings.Trim(output2, "'") == name2 {
```

Comment:
i think we can use:
```
kubectl auth can-i get --kubeconfig /etc/kubernetes/kubelet.conf --namespace kube-system configmaps/kubelet-config-x.yy
kubectl auth can-i get --kubeconfig /etc/kubernetes/kubelet.conf --namespace kube-system configmaps/kube-proxy
```

```
$ sudo kubectl auth can-i get --kubeconfig /etc/kubernetes/kubelet.conf --namespace kube-system configmaps/kube-proxy
yes
```

Reply:
do you know when this feature was introduced? oldest kubectl running this check should be 1.13

Reply:
kubectl 1.12 has it, but i don't know when it was added.